tools/chaos/cluster.go

```
@@ -192,6 +193,85 @@ func (c *Cluster) SendIngest(nodeID, source, payload string) (int, error) {
 
+// RaftKVSet sends POST /raft/kv to the named node to set a key-value pair.
+// Returns the HTTP status code (or 0 on network error).
+func (c *Cluster) RaftKVSet(nodeID, key, value string) (int, error) {
+	n := c.findNode(nodeID)
+	if n == nil {
+		return 0, fmt.Errorf("chaos: node %s not found", nodeID)
+	}
+	url := "http://" + n.Config.HTTPAddr + "/raft/kv"
+	body := fmt.Sprintf(`{"key":%q,"value":%q}`, key, value)
+	client := &http.Client{Timeout: 5 * time.Second}
+	resp, err := client.Post(url, "application/json", strings.NewReader(body))
```

golangci-lint (2.11.4)

[error] (*net/http.Client).Post must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)